scripts/parse-speakers.js

```diff
@@ -0,0 +1,138 @@
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const es = require('event-stream')
+const slug = require('slug')
+
+// Read the "cleaned" version of the speaker readme
+fs.createReadStream('./scripts/speakers-clean.md')
+  // Split at empty lines - empty lines separate the records
+  .pipe(es.split(/^$/m))
+  // Create an array from the text which contains the speaker's info
+  .pipe(es.map(function(data, callback) {
```

Comment:
Something about this area will mess up with the first entry (my name). Since my name isn't separated by a new line, it will think my name is `software-engineer`. So, that is an edge case.